_pytest/_code/code.py

```diff
@@ -110,7 +110,10 @@ def eval(self, code, **vars):
         """
         f_locals = self.f_locals.copy()
         f_locals.update(vars)
-        return eval(code, self.f_globals, f_locals)
+        try:
```

Comment:
the uses of that eval helper, `Frame.eval` should always act as eval and raise the belonging exceptions